src/core/trulens/core/database/migrations/versions/10_create_event_table.py

```diff
@@ -0,0 +1,60 @@
+"""create event table
+
+Revision ID: 10
+Revises: 9
+Create Date: 2024-12-11 09:32:48.976169
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = "10"
+down_revision = "9"
+branch_labels = None
+depends_on = None
+
+
+def upgrade(config) -> None:
+    prefix = config.get_main_option("trulens.table_prefix")
+
+    if prefix is None:
+        raise RuntimeError("trulens.table_prefix is not set")
+
+    # TODO: The automatically generated code below likely references
+    #       tables such as "trulens_feedback_defs" or "trulens_records".
+    #       However, the common prefix for these tables "trulens_" is
+    #       actually configurable and so replace it with the variable
+    #       prefix.
+    #       e.g. replace "trulens_records" with prefix + "records".
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
```

Comment:
I think it's possible to check the sqlalchemy dialect here and if so, in the case of Snowflake, can you just make an actual event table with the clustering? That way we're closer to what we want in testing.